agent/github_tools.py

```
@@ -439,7 +455,7 @@ async def get_repo(owner: str, repo: str, request: Request):
     try:
         return await gh.get_repo(owner, repo)
     except httpx.HTTPStatusError as e:
-        raise HTTPException(status_code=e.response.status_code, detail=str(e))
+        raise HTTPException(status_code=e.response.status_code, detail="Internal server error")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)